terraform/secrets.tf

```diff
@@ -0,0 +1,8 @@
+resource "aws_secretsmanager_secret" "db_password" {
+  name = "${var.env}_db_password"
+}
+
+resource "aws_secretsmanager_secret_version" "db_password" {
+  secret_id     = aws_secretsmanager_secret.db_password.id
+  secret_string = var.db_password
```

Comment:
We set the `DB_PASSWORD` in [repo's environment](https://github.com/mcagov/beacons-integration/settings/environments/195657563/edit) 

Which Github Actions passes to Terraform

Which then pops it in Secret Manager! 🎉 🎁